docs/history/changelog.md

```diff
@@ -23,6 +23,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * Add worked examples for the Map phase [#947](https://github.com/open-contracting/standard/pull/947) [#948](https://github.com/open-contracting/standard/pull/948) [#950](https://github.com/open-contracting/standard/pull/950) [#974](https://github.com/open-contracting/standard/pull/974) [#990](https://github.com/open-contracting/standard/pull/990) [#999](https://github.com/open-contracting/standard/pull/999) [#1007](https://github.com/open-contracting/standard/pull/1007).
   * Add worked examples for the Build phase [#951](https://github.com/open-contracting/standard/pull/951) [#997](https://github.com/open-contracting/standard/pull/997).
   * [#1150](https://github.com/open-contracting/standard/pull/1150) Align language in Build phase with language in Map phase.
+  * [#833](https://github.com/open-contracting/standard/issues/833) Removed language distinguishing 'core', 'community', and 'local' extensions.
```

Comment:
We have a test that checks that single-quoted words are codes from codelists. However, we don't yet perform this test on the standard repository (we do on extension repositories).

```suggestion
  * [#833](https://github.com/open-contracting/standard/issues/833) Removed language distinguishing "core", "community", and "local" extensions.
```